### DISCUSSION TITLE
No new features until launch!

### DISCUSSION THREAD

**Discussion by helgoboss** [helgoboss/helgobox#1282] [Announcements]
My highest priority now is preparing for the final launch of Playtime. Until then, I don't plan to add any new features. Just for your information. After that, I'm looking forward to dive in.

Some of the things that still need to be done before launch (or are in the works already):

- [x] Documentation (Playtime Reference)
- [x] Better ReaBoot installation process (some people experience errors)
- [x] Better error reporting if OS not compatible with Playtime
- [x] More appealing website

Thanks for your understanding and patience 🙏
Benjamin

**Comment by RibbleFripFnibbulator** [comment]
Godspeed!
